docs/HOWTO-run_tests.md

```diff
@@ -2,23 +2,27 @@ There are current two ways to run tests, using either PostgreSQL or SQLite.
 
 ## PostgreSQL Testing (default, closer to most prod environments)
 
-1) Install PostgreSQL
+1) Install PostgreSQL - [Tips to installing Postgre](https://www.codementor.io/engineerapart/getting-started-with-postgresql-on-mac-osx-are8jcopb)
 2) In PostgreSQL, create a database and user named "spoke_test":
 ```
 CREATE DATABASE spoke_test;
 CREATE USER spoke_test WITH PASSWORD 'spoke_test';
 GRANT ALL PRIVILEGES ON DATABASE spoke_test TO spoke_test;
 ```
-3) Run `npm test`
+3) Run `yarn test`
 
 ## SQLite Testing (simpler)
 
-1) Run `npm run test-sqlite`
+1) Run `yarn run test-sqlite`
+  
+## Test Redis Cache
+
```

Comment:
```suggestion
Redis is used for caching and is separate from the backend DB so can be used with sqlite *or* postgres. Redis cache testing defaults to postgres and functions like an 'addon' to the DB for improved speed/scalability.
```
Maybe add this sentence here right under this heading to clarify this section? Thanks for adding this!

Reply:
thanks for the suggestion!